src/main/java/org/alfresco/rest/core/assertion/ModelAssertion.java

```diff
@@ -309,6 +309,14 @@ public T contains(String value)
 
             return (T) model;
         }
+
+        public T notContains(String value)
+        {
+            if (fieldValue.toString().contains(value))
+                Assert.fail(errorMessage("does contain NOT expected value: " + value + ", Current Value: " + fieldValue.toString()));
```

Comment:
Shouldn't this read: `does NOT contain expected value` rather than `does contain NOT expected value`

Reply:
It's the case when in response there is e.g. aspect which was not expected. I've fixed "NOT expected" to: "unexpected".

Reply:
Makes sense now, thanks 👍